packages/remix/src/utils.ts

```diff
@@ -29,16 +31,22 @@ export function assertValidClerkState(val: any): asserts val is ClerkState | und
  * @param name
  * @returns
  */
-export const getEnvVariable = (name: string): string => {
+export const getEnvVariable = (name: string, context?: AppLoadContext): string => {
```

Comment:
@octoper  Can context be `undefined` ? I propose we change the signature to this as it seems we are always passing a parameter 
```suggestion
export const getEnvVariable = (name: string, context: AppLoadContext | undefined): string => {
```

Reply:
@panteliselef Just fixed it! There was two places we used `getEnvVariable` we didn't pass the context but I passed there also because we were never getting these variables.